score-server/README.md

```diff
@@ -44,7 +44,34 @@ bucket:
 download:
   partsize: 250000000  # 250 MB
 ```
-At this time, only a single Azure Blob Storage account (and container) is used. However, since Azure Storage can only store 500 TB per account, the Storage Server will need to manage multiple account/key credentials in the near future.  It may also make sense to have multiple containers per account as well. There are suggestions that having many objects in a single container can impose a performance penalty on some operations.
+### S3 Configuration
+To configure SCORe to use an S3-compatible object storage service, the s3 Spring profile must be activated. This profile is designed to work with services like AWS S3, MinIO, or any other compatible service. Below are the configuration properties available under the s3 section in application.yml, along with their descriptions:
+
+Profile Name: ``s3``
+
+# Configuration Properties
+
+
+s3.secured
+Description: Determines whether the connection to the S3 service should use HTTPS (true) or HTTP (false). Set to true to secure the connection.
+s3.endpoint
+Description: The URL of the S3-compatible service. This is the endpoint where the service is hosted. This property should be provided based on the service you're using.
+s3.accessKey
+Description: The access key for authenticating with the S3 service. It's required for secure access and should be kept confidential.
+s3.secretKey
+Description: The secret key paired with the access key for secure authentication to the S3 service. It must also be kept confidential.
+s3.masterEncryptionKeyId
+Description: The ID of the encryption key used for server-side encryption of files stored in S3. If provided, this key ensures that all data at rest is encrypted using the specified key.
+s3.customMd5Property
+Description:  A custom metadata property that stores an MD5 checksum of the uploaded files. This is useful for validating the integrity of the files.
+s3.connectionTimeout
+Description: The maximum amount of time (in milliseconds) the client will wait to establish a connection to the S3 service before timing out. This helps manage delays in the network or service availability.
+s3.retryLimit
+Description: The number of retries the client will attempt if an operation fails (e.g., an upload). This helps ensure robustness in case of transient issues.
+s3.sigV4Enabled
+Description: Enables AWS Signature Version 4 for request signing. This is required for certain regions or when using advanced features like server-side encryption with AWS KMS.
```

Comment:
Thank you for all this detail! Take a look at this markdown rendered by github though, it doesn't replicate empty lines the way you expect it to: https://github.com/overture-stack/score/blob/d5e506ae009afeb8b1b96cc62d9623d99420b357/score-server/README.md#configuration-properties

Let's do an editing pass here to make sure each of these properties get split across lines. Alternately, you could render this as a table with two columns, one for the property and another for the description.